server/pulp/server/async/worker_watcher.py

```diff
@@ -78,9 +78,6 @@ def handle_worker_heartbeat(event):
         msg = _("New worker '%(worker_name)s' discovered") % event_info
```

Comment:
I like the idea of the checks being distributed between all components. Would each component log warning messages when some other component is missing?